client/src/App/App.tsx

```diff
@@ -93,6 +94,7 @@ const App: React.FC = () => {
                 <ConnectedUserCredentials {...props} keycloakBaseURL={KEYCLOAK_API_BASE_URL} />
               )}
             />
+            <Route exact path="/fallback" component={Fallback} />
```

Comment:
lets also define the path string in a constants file.

Reply:
do we have to define a route for this component, I do not thin so, but I could be wrong, cc @kelvin-muchiri

Reply:
This is not required, 
Just added this for confirmation if you guys want to see the view and its working
because error boundary for sentry only work in production.

Reply:
lets leave it out.

Reply:
@HaseemUlHaq Let's add a screenshot in the PR description instead